monitors/index.js

```diff
@@ -270,7 +277,8 @@ async function Poll(callback) {
       startingBlock,
       endingBlock,
       monitorConfig,
-      uniswapPriceFeedConfig,
+      // Use uniswap if included otherwise balancer
+      uniswapPriceFeedConfig || balancerPriceFeedConfig,
```

Comment:
I think this is reasonable, prefer Uniswap by default, or use Balancer. 

If neither are passed in where do you think the best place to catch that error is? Just wondering how its dealed with right now

Reply:
yea this is a good question. I think I break the old api by requiring the user to specify one or the other. In this PR it will throw an error, but i think previously it had some uniswap defaults. I will look into that and try not to break that API.

Reply:
The uniswap price feed object is already generic (i.e. it could take any valid price feed conifg). Why not just rename this env variable to `TOKEN_PRICE_FEED_CONFIG` to signify its purpose rather than its implementation?